dotCMS/src/main/java/com/dotmarketing/business/cache/provider/h22/H22Cache.java

```diff
@@ -47,12 +44,14 @@
 
     private static final long serialVersionUID = 1L;
     final int numberOfAsyncThreads=Config.getIntProperty("cache_h22_async_threads", 10);
-    
+    final int asyncTaskQueueSize = Config.getIntProperty("cache_h22_async_task_queue", 10000);
+	final float threadAllocationTolerance = Config.getFloatProperty("cache_h22_async_tolerance",0.98F);
     final boolean shouldAsync=Config.getBooleanProperty("cache_h22_async", true);
 
 
     final ThreadFactory namedThreadFactory =  new ThreadFactoryBuilder().setNameFormat("H22-ASYNC-COMMIT-%d").build();
-    final private ExecutorService executorService = new ThreadPoolExecutor(numberOfAsyncThreads, numberOfAsyncThreads, 10, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(10000),namedThreadFactory);
+    final private LinkedBlockingQueue<Runnable> asyncTaskQueue = new LinkedBlockingQueue<>();
+    final private ExecutorService executorService = new ThreadPoolExecutor(numberOfAsyncThreads, numberOfAsyncThreads, 10, TimeUnit.SECONDS, asyncTaskQueue ,namedThreadFactory);
```

Comment:
slight optimization - should the initial thread pool be fixed at a smaller number, say 5?

Reply:
let me play with that  value a little bit see how it behaves

Reply:
looks lie we can totally afford to lower that value. I don't see that much gain in those 5 threads in terms of the number of sync calls performed